client/command/extensions/load.go

```diff
@@ -525,6 +555,20 @@ func runExtensionCmd(cmd *cobra.Command, con *console.SliverClient, args []strin
 	}
 	bd := sha256.Sum256(extdata)
 	name := hex.EncodeToString(bd[:])
+	if isBOF {
+		//if we are using a bof, we are actually calling the coffloader extension - so get the file from dep ref and use that shasum
+		dep, err := loadedExtensions[ext.DependsOn].getFileForTarget(goos, goarch)
+		if err != nil {
+			con.PrintErrorf("Dep error: %s\n", err.Error())
+			return
+		}
+		depdata, err := os.ReadFile(dep)
+		if err != nil {
+			con.PrintErrorf("dep read file error: %s\n", err)
+		}
```

Comment:
Same thing here, let's check if `depdata` is an empty slice first and error out if it is.